e2e-tests/data-sharded/run

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath "$(dirname "$0")")
+. "${test_dir}/../functions"
+
+create_namespace "$namespace"
+deploy_operator
+
+desc 'create PSMDB cluster'
+cluster="some-name"
+kubectl_bin apply \
+    -f "$conf_dir/cloud-secret.yml" \
+    -f "$conf_dir/secrets.yml" \
+    -f "$conf_dir/client.yml"
+
+apply_cluster "$test_dir/conf/$cluster.yml"
+desc 'check if all 3 Pods started'
+wait_for_running $cluster-rs0 3
+wait_for_running $cluster-cfg 3 "false"
+
+desc 'create user, enable sharding'
+
+run_mongos \
+    'db.createUser({user:"user",pwd:"pass",roles:[{db:"app",role:"readWrite"}]})' \
+    "userAdmin:userAdmin123456@$cluster-mongos.$namespace"
+sleep 2
+
+run_mongos \
+    'sh.enableSharding("app")' \
+    "clusterAdmin:clusterAdmin123456@$cluster-mongos.$namespace"
+sleep 2
+
+run_mongos \
+    'sh.shardCollection("app.city", { zipcode: 1 } )' \
+    "clusterAdmin:clusterAdmin123456@$cluster-mongos.$namespace"
+sleep 2
+
+desc 'set chunk size to 1 MB'
+run_mongos \
+    "use config\n db.settings.save( { _id:\"chunksize\", value: 1 } )" \
+    "clusterAdmin:clusterAdmin123456@$cluster-mongos.$namespace"
+sleep 2
+
+desc 'write data'
+for i in {1..100}; do
```

Comment:
loop is not needed here if `js` file is created with actual number of rows

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/526/commits/0ef77690fcb99f70a404dee504f3552e1c48a1ec